spec/plane_spec.rb

```diff
@@ -16,15 +17,40 @@
 # Are you testing that?
 
 describe Plane do
+  let(:plane) {Plane.new}
+  let(:airport) {Airport.new}
+ 	end
```

Comment:
Lint/BlockAlignment: `end` at 22, 2 is not aligned with `describe Plane do` at 19, 0